crates/router/src/services/api.rs

```diff
@@ -1316,7 +1316,22 @@ pub fn build_redirection_form(
                     }
                 }
 
-                (PreEscaped(format!("<script type=\"text/javascript\"> {logging_template} var frm = document.getElementById(\"payment_form\"); window.setTimeout(function () {{ frm.submit(); }}, 300); </script>")))
+                (PreEscaped(format!(
+                    "<script type=\"text/javascript\"> {logging_template}
```

Comment:
You can make use of raw strings, and avoid escaping the double quotes:

```rust
r#"<script type="text/javascript"> ..."#
```

Reply:
sure